public/resources/default-files/html.txt

```diff
@@ -1,15 +1,13 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Made with Thimble</title>
-</head>
-<body>
-
-  <h1>Welcome to Thimble</h1>
-
-  <p>Make something <strong>amazing</strong> with the web!</p>
-
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <title>Made with Thimble</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
```

Comment:
I'm still seeing the old version on staging, I was waiting for cloud front cache to invalidate to see if it'd automatically insert it twice or what the deal was, I think we'll have to dig into this and figure out what's the deal.